tests/console/verify_secure_boot.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify that secure boot is set as expected.
+# Maintainer: Sofia Syrianidou <ssyrianidou@suse.com>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use testapi;
+use scheduler 'get_test_suite_data';
+
+sub run {
+    my $test_data = get_test_suite_data();
+    select_console 'root-console';
+    record_info("Check file", "Check if file /sys/firmware/efi/vars/SecureBoot-*/data exists");
```

Comment:
/sys/firmware/efi/vars/SecureBoot-*/data seems to be in the testdata as well

Reply:
> /sys/firmware/efi/vars/SecureBoot-*/data seems to be in the testdata as well

not really, it is a constant, does not need to be in test_data (we could avoid hardcoded data, but it is not really necessary)

Reply:
It would make sense to me, to include it in test_data if is weren't for the wildcard *. I will keep it as it is for now.

Reply:
@jknphy it is used in more than once so it would make sense to be in a variable. As a variable of the test can also be considered testdata of the module.  

@sofiasyria wildcard is not a problem.